### DISCUSSION TITLE
web server works for dev; telnet does not

### DISCUSSION THREAD

**Discussion by gnp-x** [evennia/evennia#3536] [Evennia Questions]
In dev mode, I am able to connect to the web version via 4001, but i cannot ssh/telnet into the terminal dev version. I get the following error. I have not been able to find others with the same issue, so it might be something on my end, but unsure what it could be. Any help or guidance would be appreciated.

```sh
localhost:4000: nodename nor servname provided, or not known
```

**Comment by InspectorCaracal** [comment]
The syntax for connecting by telnet directly rather than through a MUD client usually looks more like `telnet localhost 4000` instead of the `:` syntax. It's also likely that the shell or command you're using doesn't recognize the `localhost` keyword, so try using the localhost IP of `127.0.0.1` instead

**Comment by gnp-x** [comment]
Ah it's a space... gotcha. Thanks.
